config/initializers/raven.rb

```diff
@@ -1,3 +1,5 @@
+require 'raven'
+
 if defined?(Raven) && ENV['VCAP_APPLICATION'].present?
```

Comment:
I think rather then loading Raven - we always load it anyway, it's just the `defined?` is failing because of autoloading, both the `require 'raven'` and the `defined?(Raven)` should be removed

I think this is a hangover from when Raven was only loaded in production environment but I changed it to always load Raven a while back (since with out the env var it does nothing and it then allows testing alerts in the test suite)